labels/label.go

```diff
@@ -0,0 +1,40 @@
+package main
+
+import (
+	"github.com/google/go-github/v28/github"
+)
+
+// Label respresents a GitHub label
+type Label struct {
```

Comment:
0/5 why have an internal type if `github.Label` has all the necessary fields? Can you use `*github.Label` everywhere?

Reply:
`github.Label` uses pointers to string for there field. See https://godoc.org/github.com/google/go-github/github#Label. This adds a lot of unnecessary code to `labels/mapping.go` which I wanted to avoid.

If you have good idea how to avoid casting to string pointers everywhere in  `labels/mapping.go`, let me know!